hackathon/templates/hackathon/hackathon_view.html

```diff
@@ -1,5 +1,6 @@
 {% extends "base.html" %}
 {% load static %}
+{% load my_tags %}
```

Comment:
Acceptable, but too generic perhaps?
{% load custom_tags %}
Or some sort of tag-name reference to what they're meant for?
Willing to accept this as custom_tags, though.

Reply:
Haha I did not create the file, but I agree. Renamed it to `custom_tags.py` and changed all the references to it.